Add admin prune builds and deployments command

derekwaynecarr opened the pull request:
Adds the following:

```shell
[root@openshiftdev vagrant]# openshift admin prune builds --keep-younger-than=1m
Dry run enabled - no modifications will be made.
Namespace: test, Name: ruby-sample-build-5
Namespace: test, Name: ruby-sample-build-4
Namespace: test, Name: ruby-sample-build-3
Namespace: test, Name: ruby-sample-build-2
Namespace: test, Name: ruby-sample-build-1
```

or if you really purge:

```shell
[root@openshiftdev vagrant]# openshift admin prune builds --keep-younger-than=1m --keep-failed=3 --dry-run=false
Dry run **disabled* - builds will be pruned and data will be deleted!
Namespace: test, Name: ruby-sample-build-3
Namespace: test, Name: ruby-sample-build-2
Namespace: test, Name: ruby-sample-build-1
```

/cc @ncdc @smarterclayton 

So this is functional for builds, let me know if you want to provide different output.

smarterclayton commented:
Let's do a tab writer with NAMESPACE NAME

smarterclayton commented:
If you're not dry running I don't think you need to print the dry run disabled thing.  Make sure dry run enabled is printed to stderr

derekwaynecarr commented:
Made the following changes:

1. added support to prune deployments
2. output dry run syntax only when enabled, and to stderr
3. updated display to use tab writer with NAMESPACE NAME

```shell
$ openshift admin prune builds
Dry run enabled - no modifications will be made.
NAMESPACE   NAME
test        ruby-sample-build-5
test        ruby-sample-build-4
```

smarterclayton commented:
LGTM

derekwaynecarr commented:
[merge]
